frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -1,7 +1,16 @@
-import { bucketClassNameRegex, consecutivePeriodsAndHyphensRegex } from '../constants/bucket-class';
+import {
+  bucketClassNameRegex,
+  consecutivePeriodsAndHyphensRegex,
+  timeUnits,
+} from '../constants/bucket-class';
 
 export const validateBucketClassName = (name: string): boolean =>
   name.length >= 3 &&
   name.length <= 63 &&
   bucketClassNameRegex.test(name) &&
   !consecutivePeriodsAndHyphensRegex.test(name);
+
+export const validateDuration = (ms: number): boolean => ms >= 900000 && ms <= 86400000;
+
+export const convertToMS = ({ unit, value }) =>
+  unit === timeUnits.HOUR ? parseInt(value, 10) * 3600000 : parseInt(value, 10) * 60000;
```

Comment:
Please use humanize.

Reply:
how can we convert from Hour to seconds  and minutes to millisecond ? I don't see any methods defined there to convert there. Do we have to create it ourselves ? can you post a suggestion below ...

Reply:
https://github.com/openshift/console/blob/master/frontend/public/components/utils/units.js#L47-L51
We can use this for second based manipulation. Can you add a more generic time unit and add support for hours and minutes as well.

Reply:
This not working as expected. I am getting the same results as the passed value. I think we can keep it the same for now. It is just a basic conversion.